cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -66,9 +71,33 @@ def test_appliance_scsi_control_vmware():
         casecomponent: Appliance
         caseimportance: critical
         initialEstimate: 1/4h
-    #TODO: yet to test this, once done, I will add steps. Test was not written by me originally.
     """
-    pass
+    url = ("{baseurl}/{series}/{ver}/cfme-vsphere-paravirtual-{ver}-1.x86_64.vsphere.ova"
```

Comment:
This image name pattern should also be pulled from `cfme_data`, or `miq_version`. It is 100% related to downstream images, since paravirtual images aren't built for upstream.

The test needs to ignore upstream, as it won't work - the `series` is master.